src/pickup.cpp

```diff
@@ -169,6 +181,50 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
             veh_tool( "hotplate" );
             return DONE;
 
+        case USE_WASHMACHINE: {
+            bool detergent_is_enough = false;
+            bool filthy_items = false;
+            if( g->u.crafting_inventory().has_charges( "detergent", 5 ) ) {
+                detergent_is_enough = true;
```

Comment:
Is that too long?
`bool detergent_is_enough = g->u.crafting_inventory().has_charges( "detergent", 5 );`